### ISSUE TITLE
getaddrinfo ENOENT in offline Mode

### ISSUE THREAD

**Issue by shanalikhan** [Microsoft/vscode#12183]
- VSCode Version: 1.5.2
- OS Version: Win 10

Steps to Reproduce:

1. Start the editor in offline mode , no internet connected
2. and error pop up.

showing information message : `getaddrinfo ENOENT marketplace.visualstudio.com:443`
```
messageService.ts:119getaddrinfo ENOENT marketplace.visualstudio.com:443: Error: getaddrinfo ENOENT marketplace.visualstudio.com:443
    at Object.exports._errnoException (util.js:890:11)
    at errnoException (dns.js:31:15)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:77:26)e.doShow @ messageService.ts:119
threadService.ts:217[Extension Host] Windows_NT
```

**Comment by joaomoreno** [comment]
This is [fixed](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/extensions/electron-browser/extensionsWorkbenchService.ts#L525) in `master`.

**Comment by shanalikhan** [comment]
Its still appearing in 1.5.3

**Comment by joaomoreno** [comment]
Fix comes with 1.6.
